sdks/java/core/src/main/java/org/apache/beam/sdk/values/BeamRecordType.java

```diff
@@ -49,22 +49,22 @@ public void validateValueType(int index, Object fieldValue)
     * Get the coder for {@link BeamRecordCoder}.
     */
    public BeamRecordCoder getRecordCoder(){
-     return BeamRecordCoder.of(this, fieldsCoder);
+     return BeamRecordCoder.of(this, fieldCoders);
    }
 
-   public List<String> getFieldsName(){
-     return fieldsName;
+   public List<String> getFieldNames(){
```

Comment:
Looking at the other names below, it occurs to me that field names are just being referred to as fields. We should be consistent, and either call this one getFields(), or else rename those getFieldNameByIndex, etc.